earth_enterprise/src/fusion/portableglobe/gerewritedbroot.cpp

```diff
@@ -187,16 +187,22 @@ void ReplaceSearchServer(const std::string &search_service,
  */
 void ReplaceReferencedKml(const std::string &new_kml_base_url,
                           const std::string &kml_url_to_file_map_file,
+                          const bool preserve_kml_filenames,
```

Comment:
Need to update the header comment to this function for the new field.

Reply:
Done.